src/Unosquare.PassCore.Web/Controllers/PasswordController.cs

```diff
@@ -43,6 +52,42 @@ public class PasswordController : Controller
         [HttpGet]
         public IActionResult Get() => Json(_options);
 
+        /// <summary>
+        /// Returns generated password as a JSON string.

```

Comment:
This is a really insecure passphrase generator.

If you're going to create human readable/memorable passwords you should use an established and well reviewed algorithm: https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki#Generating_the_mnemonic

Reply:
@Serk352 this looks like a good library https://github.com/elucidsoft/dotnetstandard-bip39

Reply:
I don't know if we should add BIP39. Maybe keep the current generator and alternative fixed-length random char generator. What do you think?

Reply:
The current generator is insecure and encourages bad passwords, whatever choice is made about BIP39 the current generator must be removed